src/index.ts

```diff
@@ -137,6 +148,32 @@ export class WaxJS {
     return this.user.account;
   }
 
+  private async getChainInfoByChainName(chainName: string): Promise<any> {
+    const response = await this.rpc.get_table_rows({
+      json: true,
+      code: "registry.wax",
+      scope: "registry.wax",
+      table: "chains",
+    });
+    const chainInfo: any | null = response.rows.find(row => row.chain_name === chainName);
+
+    if (!chainInfo) {
+      // TODO: handle case chain info not exists
+    }
+
+    return chainInfo;
+  }
+
+  public async setChainName(chainName: string): Promise<void> {
+    const chainInfo = await this.getChainInfoByChainName(chainName);
+
+    if (!chainInfo) return;
```

Comment:
I think this should throw an exception. We don't want developers to set a bad chain name and think it worked

Reply:
updated base on comment 👍